src/status_im2/contexts/quo_preview/main.cljs

```diff
@@ -105,6 +106,9 @@
                            {:name      :dynamic-button
                             :insets    {:top false}
                             :component dynamic-button/preview-dynamic-button}]
+   :cards                 [{:name      :small-option-card
```

Comment:
this should be :on-boarding it's mapped to the Figma design file to keep it familiar to navigate

Reply:
OK! it's solved in `423830483381af610ae3284b84049029405b5c40`